index.js

```diff
@@ -5,25 +5,33 @@ const raven = require('raven');
 const sentryStream = require('bunyan-sentry-stream');
 const le = require('le_node');
 
-// trace is a useful default for dev
 const loggerLevel = process.env.LOGGER_LEVEL || 'info';
 const loggerName = process.env.LOGGER_NAME || 'Developpment logger';
 
-// Default logger configuration:
 const config = {
   name: loggerName,
   level: loggerLevel,
-  streams: [{
-    level: loggerLevel,
-    stream: process.stdout
-  }],
+  streams: [],
   serializers: {
-    err: bunyan.stdSerializers.err   // <--- use this
+    err: bunyan.stdSerializers.err
   }
 };
 
+if (process.env.NODE_ENV === 'test') {
```

Comment:
you should never do this kind of check. Define another env variable that is set by your test setup file